java/ql/lib/ext/generated/io.jsonwebtoken.model.yml

```diff
@@ -0,0 +1,498 @@
+      - ["io.jsonwebtoken", "PrematureJwtException", true, "PrematureJwtException", "(Header,Claims,String)", "", "Argument[1].Element", "Argument[this]", "taint", "df-generated"]
+      - ["io.jsonwebtoken", "PrematureJwtException", true, "PrematureJwtException", "(Header,Claims,String)", "", "Argument[2]", "Argument[this]", "taint", "df-generated"]
+      - ["io.jsonwebtoken", "PrematureJwtException", true, "PrematureJwtException", "(Header,Claims,String,Throwable)", "", "Argument[0].Element", "Argument[this]", "taint", "df-generated"]
+      - ["io.jsonwebtoken", "PrematureJwtException", true, "PrematureJwtException", "(Header,Claims,String,Throwable)", "", "Argument[1].Element", "Argument[this]", "taint", "df-generated"]
+      - ["io.jsonwebtoken", "PrematureJwtException", true, "PrematureJwtException", "(Header,Claims,String,Throwable)", "", "Argument[2]", "Argument[this]", "taint", "df-generated"]
+      - ["io.jsonwebtoken", "PrematureJwtException", true, "PrematureJwtException", "(Header,Claims,String,Throwable)", "", "Argument[3]", "Argument[this]", "taint", "df-generated"]
+      - ["io.jsonwebtoken", "RequiredTypeException", true, "RequiredTypeException", "(String)", "", "Argument[0]", "Argument[this]", "taint", "df-generated"]
+      - ["io.jsonwebtoken", "RequiredTypeException", true, "RequiredTypeException", "(String,Throwable)", "", "Argument[0]", "Argument[this]", "taint", "df-generated"]
+      - ["io.jsonwebtoken", "RequiredTypeException", true, "RequiredTypeException", "(String,Throwable)", "", "Argument[1]", "Argument[this]", "taint", "df-generated"]
+      - ["io.jsonwebtoken", "SignatureException", true, "SignatureException", "(String)", "", "Argument[0]", "Argument[this]", "taint", "df-generated"]
+      - ["io.jsonwebtoken", "SignatureException", true, "SignatureException", "(String,Throwable)", "", "Argument[0]", "Argument[this]", "taint", "df-generated"]
+      - ["io.jsonwebtoken", "SignatureException", true, "SignatureException", "(String,Throwable)", "", "Argument[1]", "Argument[this]", "taint", "df-generated"]
+      - ["io.jsonwebtoken", "UnsupportedJwtException", true, "UnsupportedJwtException", "(String)", "", "Argument[0]", "Argument[this]", "taint", "df-generated"]
+      - ["io.jsonwebtoken", "UnsupportedJwtException", true, "UnsupportedJwtException", "(String,Throwable)", "", "Argument[0]", "Argument[this]", "taint", "df-generated"]
+      - ["io.jsonwebtoken", "UnsupportedJwtException", true, "UnsupportedJwtException", "(String,Throwable)", "", "Argument[1]", "Argument[this]", "taint", "df-generated"]
+      - ["java.util", "Map", true, "put", "(String,Object)", "", "Argument[0]", "Argument[this]", "taint", "df-generated"]
+      - ["java.util", "Map", true, "put", "(String,Object)", "", "Argument[1]", "Argument[this]", "taint", "df-generated"]
+      - ["java.util", "Map", true, "put", "(String,Object)", "", "Argument[this]", "ReturnValue", "taint", "df-generated"]
```

Comment:
I didn't realize from the discussion elsewhere that these models are from `java.util`, not `io.jsonwebtoken`. It seems potentially odd that the model generator is including these when creating models for the `io.jsonwebtoken` framework. I wonder if it's worth looking into this discrepancy further?

Reply:
These should definitely not be added.